--- gok-0.9.3/Makefile.in.wiget	2003-11-16 14:57:52.000000000 +0100
+++ gok-0.9.3/Makefile.in	2003-11-16 14:58:47.000000000 +0100
@@ -659,12 +659,12 @@
 	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) gconftool-2 --makefile-install-rule $(schema_DATA)
 	@for l in *; do 						\
 		if [ -f $$l/main.kbd ]; then 				\
-			if [ ! -d $(datadir)/gok/$$l ]; then 		\
-				mkdir $(datadir)/gok/$$l; 		\
+			if [ ! -d $(DESTDIR)$(datadir)/gok/$$l ]; then 		\
+				mkdir $(DESTDIR)$(datadir)/gok/$$l; 		\
 			fi;						\
 			for f in $$l/*.kbd; do				\
-				echo copying $$f to $(datadir)/gok/$$l;	\
-				cp $$f $(datadir)/gok/$$l; 		\
+				echo copying $$f to $(DESTDIR)$(datadir)/gok/$$l;	\
+				cp $$f $(DESTDIR)$(datadir)/gok/$$l; 		\
 			done;						\
 		fi;							\
 	done
