diff -urN gok-0.11.9.orig/gok.desktop.in gok-0.11.9/gok.desktop.in
--- gok-0.11.9.orig/gok.desktop.in	2004-10-01 15:34:58.441534720 +0200
+++ gok-0.11.9/gok.desktop.in	2004-10-01 15:35:23.271759952 +0200
@@ -6,7 +6,7 @@
 Terminal=false
 Type=Application
 Icon=gok.png
-Categories=Application;Accessibility;
+Categories=GTK;GNOME;Accessibility;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gok
 X-GNOME-Bugzilla-Component=general
