diff -urN aa/gok-0.11.10/gok/gok-keyboard.c gok-0.11.10/gok/gok-keyboard.c
--- aa/gok-0.11.10/gok/gok-keyboard.c	2004-10-06 16:57:47.000000000 +0200
+++ gok-0.11.10/gok/gok-keyboard.c	2004-10-07 19:09:13.420154120 +0200
@@ -3950,7 +3950,6 @@
 	GokOutput    delim_output;
 	GokKeyboard* pPredictedKeyboard;
 	GokKeyboard* pKeyboardTemp;
-	GokKeyboard* keyboard = gok_main_get_current_keyboard ();
 	GokKey* pKeyTemp;
 	const gchar *str;
 	
