diff -urN aa/gok-0.9.4/Makefile.am gok-0.9.4/Makefile.am
--- aa/gok-0.9.4/Makefile.am	2003-12-08 11:25:23.000000000 +0100
+++ gok-0.9.4/Makefile.am	2003-12-28 16:52:21.012098480 +0100
@@ -94,6 +94,9 @@
 
 install-data-local:
 	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) gconftool-2 --makefile-install-rule $(schema_DATA)
+	if [ ! -d $(DESTDIR)$(datadir)/gok ]; then		\
+		mkdir $(DESTDIR)$(datadir)/gok;		\
+	fi;
 	@for l in *; do 						\
 		if [ -f $$l/main.kbd ]; then 				\
 			if [ ! -d $(DESTDIR)$(datadir)/gok/$$l ]; then 		\
